src/listener/server-profile-templates.js

```diff
@@ -46,7 +46,7 @@ class ServerProfileTemplateListener extends Listener {
     this.respond(this.LIST_DEPLOYED_PROFILES, this.GetDeployedProfiles.bind(this));
     this.capabilities.push(this.BULLET + "Show profile(s) using a server profile template (e.g. show profiles using docker swarm).");
 
-    this.CREATE_PROFILES_FROM_TEMPATE = /(?:deploy|create) (:<count>\d+) profile[s]{0,1} (?:from|for|using) (:<host>.*?)(?:\/rest\/server-profile-templates\/)(:<templateId>[a-zA-Z0-9_-]*?)\.$/i;
+    this.CREATE_PROFILES_FROM_TEMPATE = /(?:deploy|create) a profile (?:from|for|using) (:<host>.*?)(?:\/rest\/server-profile-templates\/)(:<templateId>[a-zA-Z0-9_-]*?)\.$/i;
     this.respond(this.CREATE_PROFILES_FROM_TEMPATE, this.DeployProfiles.bind(this));
     this.capabilities.push(this.BULLET + "Create profile(s) using a server profile template (e.g. create 1 profile from docker swarm).");
```

Comment:
`Create profile` instead of `Create profile(s)`.

Reply:
(e.g. create **a** profile from docker swarm).

Reply:
Good catch thanks!